### DISCUSSION TITLE
Temperatures

### DISCUSSION THREAD

**Discussion by StarAsus123** [seerge/g-helper#2484] [General]
Hello. I would like to ask about my laptop. I have Asus rog scar 17 , rtx 4070. My temperature at gaming is 90+ for cpu and 70-85 for gpu . I play every game for example from 2014, 2018, and modern like 2023, and 2022 and situation always same, it doesn't matter hie tough game is , laptop hot always... I put performance mode and fans are loud and I am kinda disappointed that laptop always hot also I am worried how long this laptop can work if it's always 95 degree because as I know this is critical temperature,  does it mean that my laptop will broke within one year for example? Sometimes gpu also 85, jumps to 87, 88. So I would like to know if anybody have the same problem? Also my laptop Sometimes doesn't wake up after sleeping mode, screen is black even when I press power button or keys . And sometimes laptop restart while gaming when it's hot. It happens while Shadow of mordor the game which is 2014 year release.

**Comment by StarAsus123** [comment]
I use armory crate

**Comment by seerge** [comment]
@StarAsus123 

> I am worried how long this laptop can work if it's always 95 degree because as I know this is critical temperature

That's **not true**. 96C is just a "throttle temp" (i.e. after reaching that CPU will lower it's own consumption in order to stay cooler).

If you are so worried about CPU temperatures during gaming, you can just set a limit around 35-40W under ``Fans+Power`` and click apply. That's usually enough for gaming.

**Comment by StarAsus123** [comment]
I have another question about fps in games. I launched far cry 4 on ultra settings, and my frame rate changes too much, on average from 70 to 90. But during the game, fps can jump from 90 to 80 or from 120 to 100, 90 this happens quickly, then returns back , then while moving around the map, fps will jump to 60, then to 75 and so on. The main problem is that at this second there is a slight delay, this delay is almost unnoticeable, it comes from such a dynamic FPS. And I don't know how to make fps more stable. Graphics settings are done automatically via Nvidia geforce experience. The processor is at 95 degrees. I set my settings to armory crate to 40 watts and the processor temperature became about 80-82. But these delays have not disappeared. I'm wondering if this is a problem with the processor due to throttling or if I need to change something in the Nvidia control panel. Because I don’t understand how such a video card can’t handle such an old game..
